control-plane/api/v1alpha1/shared_types.go

```diff
@@ -310,6 +310,35 @@ func (in *FailoverPolicy) validate(path *field.Path) field.ErrorList {
 	return errs
 }
 
+type PrioritizeByLocality struct {
```

Comment:
Just to conform with the other structs in this file, can you add a comment for this public struct?

Reply:
Ah, thanks for catching! Absolutely.